### PR TITLE
Book store irasp

### PR DESCRIPTION
**Pull request by Irasp11**

### CODE REVIEW

File: src/test/java/api/account/Account.java [hunk 1]
```diff
@@ -2,40 +2,49 @@
...
-}
 
+    public Response deleteAccount(RegisterModelDto user) {
+        String endPoint = "/Account/v1/Authorized";
+        return postRequest(endPoint, 404, user);
+    }
+
+    public RegisterModelDto createNewRandomAccount() {
+        RegisterModelDto register = new RegisterModelDto();
+        register.setUserName(faker.internet().uuid());
```
Comment (LeonadO555) [inline src/test/java/api/account/Account.java]: faker.internet().uuid() need to move in variable

File: src/test/java/schemas/RegisterModelDto.java [hunk 2]
```diff
@@ -11,60 +10,11 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @JsonInclude(JsonInclude.Include.NON_DEFAULT)
-@JsonIgnoreProperties(ignoreUnknown = true)
```
Comment (LeonadO555) [inline src/test/java/schemas/RegisterModelDto.java:14]: need to rollback this changes

File: src/test/java/e2e/pages/ProfilePage.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+//            goToStoreButton.click();
+//        } catch (NoSuchElementException e) {
+//            System.out.println("Element not found : " + e.getMessage());
+//        }
+//    }
+//}
+    public void clickGoToBookStoreButton() {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        try {
+            WebElement goToStoreButton = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//*[@id='gotoStore']")));
```
Comment (LeonadO555) [inline src/test/java/e2e/pages/ProfilePage.java]: need to describe waitForLoading and move to waitForLoading method all waiting options (forClickble, forVisible)

File: src/test/java/api/account/Account.java (hunk 1 above)
Comment (Irasp11) [reply src/test/java/api/account/Account.java]: made

File: src/test/java/schemas/RegisterModelDto.java (hunk 2 above)
Comment (Irasp11) [reply src/test/java/schemas/RegisterModelDto.java:14]: made

File: src/test/java/e2e/pages/ProfilePage.java (hunk 3 above)
Comment (Irasp11) [reply src/test/java/e2e/pages/ProfilePage.java]: made

**Review by LeonadO555 — APPROVED** [review]

**Pull request merged**